### DISCUSSION TITLE
WebView runtime version requirements?

### DISCUSSION THREAD

**Discussion by kyle-mcknight** [airsdk/Adobe-Runtime-Support#2609] [Q&A]
What is the minimum version of the WebView2 runtime that must be installed on the end-user's machine?

Thank you

**Comment by ajwfrost** [comment]
We should be able to cope with Edge WebView2 91.0.824.0 or later (1.0.824 is the version of the WebView2 SDK) - it may work with earlier ones too, but some of the features we use hit 'stable' in that release. This version is around 2 years old now..

thanks
